### ISSUE TITLE
Non-descriptive long link out of context in GOV.UK Review application

### ISSUE THREAD

**Issue by nickcolley** [alphagov/govuk-frontend#1402]
This issue is from a May 2019 external accessibility audit report.

**WCAG Reference**: [2.4.9 Link Purpose (Link Only) (Level AAA)](https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-link-only.html)
**Issue ID**: DAC_Issue18
**URL**: http://govuk-frontend-v3.herokuapp.com/full-page-examples/check-your-answers#/name
**Page**: Check your answers

## Summary

Some links were found to not be descriptive to users that navigate out of the context of the page

## Screen Shots
![](https://user-images.githubusercontent.com/2445413/58711563-b1b18b00-83b6-11e9-8e7f-64b67aae7df1.png)
![](https://user-images.githubusercontent.com/2445413/58711567-b413e500-83b6-11e9-92c0-0ebb33d5caf8.png)

There was a link on the page that was not descriptive to users that navigate out of the context of the page and would benefit from more description. Although this is a template URL it is worth noting that descriptive link text should be used in place of URLs.

## Current Code Ref(s)
```html
<a class="govuk-link" href="#">https://reallylongurlformyeventthatneedsatemporaryeventsnotice.com</a>
```

## Screen reader comments
> “While navigating both in and out of context I located a link which possessed a link text of a URL. It was not immediately clear to me what the link related to as I needed to read the web address. It would assist me if the link text could possess full phrases to indicate the purpose of the link rather than a URL. This issue occurred using JAWS, NVDA and VoiceOver for iPhone and Mac.”

## Solution
Ensure that descriptive text is used instead of URLs as these do not make sense to users that navigate out of the context of the page.

### Example

Find out more about`<a class="govuk-link" href=" https://reallylongurlformyeventthatneedsatemporaryeventsnotice.com"> Temporary Event Notice (TEN) </a>`

**Comment by nickcolley** [comment]
I have assigned this low priority since this link is only in our GOV.UK Frontend review application.

The link is intentionally long to check the layout of the summary list component, there maybe a better example that is also usable but if we cannot think of something I'd recommend we accept this as a failure and close this issue.

**Comment by kellylee-gds** [comment]
Remove it

**Comment by nickcolley** [comment]
Confirmed as fixed in a re-test by DAC on 7th October 2019
